app/assets/stylesheets/pageflow/themes/legacy/overview.scss

```diff
@@ -0,0 +1,48 @@
+.overview {
+  @import "pageflow/themes/default/overview/chapter_border_images";
+
+  font-family: sourcesanspro;
+
+  &:after {
+    background-color: black;
+  }
+
+  &.active:after {
+    opacity: 0.4;
+  }
+
+  .content {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.